src/vunnel/providers/rocky/client.py

```diff
@@ -0,0 +1,49 @@
+import logging
+import os
+from collections.abc import Generator
+from pathlib import Path
+
+import orjson
+
+from vunnel.utils import http_wrapper
+
+
+class Client:
+    _default_api_path_ = "/api/v3/osv/"
+
+    def __init__(
+        self,
+        download_path: str,
+        logger: logging.Logger | None = None,
+        rocky_versions: list[str] | None = None,
+        api_host: str = "https://apollo.build.resf.org",
+    ):
+        if rocky_versions is None:
+            rocky_versions = ["8", "9"]
```

Comment:
is there a way to make this more dynamic? Some providers hard code this kind information either because it's not possible to dynamically figure it or (most of the time) we didn't want to have enterprise deployments start failing when unexpected data started flowing from a new version. The latter point is no longer a risk. 

Consider this comment not a request to change anything now but more on how we can improve these kinds of case systematically in the future (may be worth an issue too).

Reply:
It turns out this wasn't doing anything, so I just removed it.